pages/linux/sfill.md

```diff
@@ -0,0 +1,24 @@
+# sfill
+
+> Secure overwrite free space and inodes of the partition where the specified directory resides.
+> More information: <https://manned.org/sfill>.
+
+- Overwrite free space and inodes of an USB with 38 writes (slow but secure):
```

Comment:
Why is a "USB" mentioned specifically here? Does it not work with other kinds of disks?/

Reply:
It also works for other (kinds of) disks, I think people mainly use this to securly clean up USB drives.

Reply:
Correct, mainly with USBs or external disks as far as i know. The main description ("..of the partition where the specified directory resides..") is indeed more generic but I preferred to use a more useful example.

Reply:
I say it should be `a USB` because I think it's generally pronouced as the three consecutive letters `U S B`.  Thoughts, anyone?  This might also be a regional thing, not sure.
```suggestion
- Overwrite free space and inodes of a USB with 38 writes (slow but secure):
```

Reply:
@bl-ue you're right, 'a USB' is the most correct form. Fixed.

Reply:
Hrm. I would prefer `a disk` to "a USB". It's more correct to how the command actually works and sounds better too.

Reply:
Changed